lab4/viewpoints/development/development.tex

```diff
@@ -0,0 +1,131 @@
+\section{Development View}
+
```

Comment:
Description of our usage of ASP.NET. Usage of dependency injection. In functional viewpoint I described layered structure, better module organization is to have Interface module, business logic (also referred as Core module) and data access layer.